<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateStation</class>
 <widget class="QDialog" name="UpdateStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="workArea">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>请选择工作区域</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>新增工位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改工位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除工位</string>
   </property>
  </widget>
  <widget class="QTableView" name="stationTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>641</width>
     <height>301</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>工作区域：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>工位类型等候区:请填写英文WAIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>后备区:请填写英文READY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>工位类型充电区:请填写英文RECHARGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>“1”表占用，“0”表空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
